src/charm.py

```diff
@@ -278,6 +307,13 @@ def validate_exporter_configs(self) -> Tuple[bool, str]:
             )
             return False, "Invalid config: 'exporter-log-level'"
 
+        redfish_creds = self.get_redfish_creds()
+        if redfish_creds:
+            redfish_creds_valid = self.validate_redfish_creds(redfish_creds)
+            if not redfish_creds_valid:
+                logger.error("Invalid redfish credentials.")
+                return False, "Invalid config: 'redfish-username' or 'redfish-password'"
+
```

Comment:
If you agree with my suggestion above this can be reduced to
```suggestion
        if self.validate_redfish_creds is False:
            logger.error("Invalid redfish credentials.")
            return False, "Invalid config: 'redfish-username' or 'redfish-password'"

```

Reply:
Done